content/tutorials/build-apps/build-a-desktop-wallet-in-javascript.md

```diff
@@ -0,0 +1,1433 @@
+```
+
+6. In `view/renderer.js` add at the top:
+
+```javascript
+window.electronAPI.onOpenSeedDialog((_event) => {
+    const seedDialog = document.getElementById('seed-dialog');
+    const seedInput = seedDialog.querySelector('input');
+    const submitButton = seedDialog.querySelector('button[type="submit"]');
+
+    submitButton.addEventListener('click', () => {
+        const seed = seedInput.value;
+        window.electronAPI.onEnterSeed(seed)
+        seedDialog.close()
+    });
+
+    seedDialog.showModal()
+})
+
+window.electronAPI.onOpenPasswordDialog((_event) => {
+    const passwordDialog = document.getElementById('password-dialog');
+    const passwordInput = passwordDialog.querySelector('input');
+    const submitButton = passwordDialog.querySelector('button[type="submit"]');
+
+    submitButton.addEventListener('click', () => {
+        const password = passwordInput.value;
+        window.electronAPI.onEnterPassword(password)
+        passwordDialog.close()
+    });
+
+    passwordDialog.showModal()
+});
+```
+
+Start up the application:
+
+```console
+npm run start
+```
+
+On first run, It should first prompt you for an account seed and then for a password. After you have created a wallet this way, you should close the application and start it up a second time: On second run it should prompt you for the password, and you should see the same result as in the last step.
+
+To run the reference application found in `content/_code-samples/build-a-wallet/desktop-js` for this step, run:
+
+```console
+npm run password
+```
+
+### 6. Styling
+
+**Full code for this step:**
+[`6_styling.js`]({{target.github_forkurl}}/tree/{{target.github_branch}}/content/_code-samples/build-a-wallet/desktop-js/6_styling.js).
+[`library/3_helper.js`]({{target.github_forkurl}}/tree/{{target.github_branch}}/content/_code-samples/build-a-wallet/desktop-js/library/3_helper.js).
+[`library/4_helper.js`]({{target.github_forkurl}}/tree/{{target.github_branch}}/content/_code-samples/build-a-wallet/desktop-js/library/4_helper.js).
+[`library/5_helper.js`]({{target.github_forkurl}}/tree/{{target.github_branch}}/content/_code-samples/build-a-wallet/desktop-js/library/5_helper.js).
+[`view/6_preload.js`]({{target.github_forkurl}}/tree/{{target.github_branch}}/content/_code-samples/build-a-wallet/desktop-js/view/6_tx-preload.js).
+[`view/6_styling.html`]({{target.github_forkurl}}/tree/{{target.github_branch}}/content/_code-samples/build-a-wallet/desktop-js/view/6_styling.html).
+[`view/6_renderer.js`]({{target.github_forkurl}}/tree/{{target.github_branch}}/content/_code-samples/build-a-wallet/desktop-js/view/6_tx-renderer.js).
+
+After finishing this step the application should look like this:
+
+![Screenshot: Step 6, style application with css](img/javascript-wallet-6.png)
+
+1. Copy the folder `bootstrap` and its contents to your project directory. 
+
+2. Also, copy the file `view/custom.css` as well as `XRPLedger_DevPortal-white.svg` to the `view`directory.
```

Comment:
These should be two steps, and should have links to the GitHub repo

Reply:
Also, `view/custom.css` does not exist anymore. It's now just `bootstrap/custom.css` it seems.

Reply:
Instead of saying 'copy' - this step should also probably say 'move' since the custom.css file needs to moved out of the bootstrap folder into the view folder to work.

Reply:
Bumping this issue

Reply:
So, this is now all in the bootstrap directory, the description now has a proper link, narrowed down to two steps

Reply:
Looks good :)